nucypher/blockchain/eth/agents.py

```diff
@@ -984,6 +985,7 @@ def fetch_policy(self, policy_id: bytes) -> PolicyInfo:
     def create_policy(self,
                       policy_id: bytes,
                       transacting_power: TransactingPower,
+                      size: int,
```

Comment:
To remain consistent, should we use `shares` instead of `size`. It's currently split between the python side (eg. payment.py) which uses `shares` and the solidity contract which uses `size` - the agent is caught in the middle heh